Karma/Controllers/AdminController.cs

```diff
@@ -342,5 +344,79 @@ private void hideReportedPost(int id)
             post.State = (int)Post.PostState.Hidden;
             post.IsVisible = false;
         }
+
+        [HttpGet]
+        //[ValidateAntiForgeryToken]
+        public async Task<IActionResult> ViewReports(int? page, string sorting, string? aggr)
```

Comment:
This still needs to be divided to smaller units, but I'll wait for that in the next PR, like we talked.

Thanks for the changes.

Reply:
Yes, working on it.